.github/workflows/web.yml

```
@@ -83,6 +89,25 @@ jobs:
         with:
           features_path: centreon/tests/e2e/features
 
+  dispatch-to-maintained-branches:
+    if: ${{ github.event_name == 'schedule' && github.ref_name == 'develop' }}  ||
+      github.event_name == 'pull_request' && github.event.pull_request.title ||
+      github.event_name == 'push' && github.event.head_commit.message ||
+      github.event_name == 'workflow_dispatch' && github.workflow || 'unknown' }}
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      - run: |
+          gh workflow run web.yml -r "dev-23.10.x"
+          gh workflow run web.yml -r "dev-23.04.x"
+          gh workflow run web.yml -r "dev-22.10.x"
+          gh workflow run web.yml -r "dev-22.04.x"
```

you could add an boolean input to the workflow which defined if it is a nightly build
this should be false by default
(environment input may be used also)
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_dispatchinputs
```suggestion
          gh workflow run web.yml -r "dev-23.10.x" -f is_nightly=true
          gh workflow run web.yml -r "dev-23.04.x" -f is_nightly=true
          gh workflow run web.yml -r "dev-22.10.x" -f is_nightly=true
          gh workflow run web.yml -r "dev-22.04.x" -f is_nightly=true
```